CHANGELOG.md

```diff
@@ -3,6 +3,113 @@
 Subscribe to the [tracking issue #2231](https://github.com/tigerbeetle/tigerbeetle/issues/2231)
 to receive notifications about breaking changes!
 
+## TigerBeetle 0.16.0
```

Comment:
As per our new release process, you should merge new additions with the past unreleased entries:

https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/about/internals/releases.md#skipping-release

We don't want require our users to piece together the actual changelog!

Reply:
Makes sense - I missed that last sentence when checking out the diff!